src/main/java/de/tum/in/www1/artemis/service/QuizSubmissionService.java

```diff
@@ -194,19 +192,27 @@ private void checkSubmissionForLiveModeOrThrow(Long exerciseId, String userLogin
     }
 
     /**
-     * Updates a submission for the exam mode
+     * Returns true if student has submitted at least once for the given quiz batch
      *
-     * @param quizExercise   the quiz exercise for which the submission for the exam mode should be done
-     * @param quizSubmission the quiz submission includes the submitted answers by the student
-     * @param user           the student who wants to submit the quiz during the exam
-     * @return the updated quiz submission after it has been saved to the database
+     * @param quizBatch the quiz batch of interest to check if submission exists
+     * @param login     the student of interest to check if submission exists
+     * @return boolean the submission status of student for the given quiz batch
      */
-    public QuizSubmission saveSubmissionForExamMode(QuizExercise quizExercise, QuizSubmission quizSubmission, User user) {
-        // update submission properties
-        quizSubmission.setSubmitted(true);
-        quizSubmission.setType(SubmissionType.MANUAL);
-        quizSubmission.setSubmissionDate(ZonedDateTime.now());
+    public boolean isSubmitted(QuizBatch quizBatch, String login) {
```

Comment:
Maybe rename to `hasUserSubmitted`. I dont know if login as a String parameter makes sense. Is there no way to use user instead?

Reply:
yes, hasUserSubmitted is more clear. But the login parameter is required as the information is expected to be defined by the caller.